src/sentry/static/sentry/app/views/usageStats/usageChart/index.tsx

```diff
@@ -61,8 +61,8 @@ export const CHART_OPTIONS_DATA_TRANSFORM: SelectValue<ChartDataTransform>[] = [
     disabled: false,
   },
   {
-    label: t('Day-to-Day'),
-    value: ChartDataTransform.DAILY,
+    label: t('Periodic'),
```

Comment:
In the subscription chart we call it "Day-to-day" -- should it be standardize?

Reply:
or is this different than day-to-day?

Reply:
For subscription, a bar is always equivalent to 1 day. For usage stats, a bar could represent 1 hour, 4 hours, etc. 

`periodic` is an alright 1-word description for that, but I can change it if there's something better

Reply:
ah right. okay sounds good